src/lib/dev-environment/dev-environment-core.js

```diff
@@ -310,12 +323,14 @@ async function prepareLandoEnv( instanceData, instancePath ) {
 	const nginxFileTargetPath = path.join( nginxFolderPath, nginxFileName );
 	const instanceDataTargetPath = path.join( instancePath, instanceDataFileName );
 
-	fs.mkdirSync( instancePath, { recursive: true } );
-	fs.mkdirSync( nginxFolderPath, { recursive: true } );
+	await fs.promises.mkdir( instancePath, { recursive: true } );
```

Comment:
Why do you prefer the async api here?

I think `fs.promises.mkdir` is supported on less node versions than `fs.mkdirSync`.

Reply:
https://github.com/Automattic/vip-cli/blob/develop/package.json#L77

The Promises API is available since [v10.0.0](https://nodejs.org/api/fs.html#promises-api).

As for the async API - it probably does not matter much for a CLI application, but in general, it is [not recommended to block the event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/).

With the async API, we can use [`Promise.all`](https://github.com/Automattic/vip-cli/pull/1149/files/f4edb944f3d17d9e04a829177a6106a9ac0e8f87#diff-c7d8beaf4cbe55eeb49f1a4ae46c4b0efd1ea6f21d1863eeea06554e89bbc57dR329-R333) to make things a bit faster.

Reply:
Oh right:
> VIP-CLI requires a minimum installed version of Node.js v14 and npm v6
From the [docs](https://docs.wpvip.com/technical-references/vip-cli/installing-vip-cli/#h-update-node-js-and-npm)

So we should be all good here.